tests/test_soak.py

```diff
@@ -101,6 +103,7 @@
 
 
 def setup_module(module):
+    sdk_cmd.run_raw_cli("package install {} --yes --cli".format(utils.SPARK_PACKAGE_NAME))
     if not shakedown.package_installed('spark', SOAK_SPARK_SERVICE_NAME):
         additional_options = {
             "hdfs": {
```

Comment:
The `config-url` is incorrect for the soak clusters where HDFS is unstalled under `/data-services/hdfs`.